src/objects/metricsEvent.ts

```diff
@@ -197,9 +197,11 @@ export const toErrorMetricsEvent = (e: any, tag: string, apiId: NodeApiIds): Eve
       case statusCode == 400:
         return createBadRequestErrorMetricsEvent(tag, apiId);
       case statusCode == 401:
-        return createUnauthorizedErrorMetricsEvent(tag, apiId);
+        console.log('An unauthorized error occurred. Please check your API Key.');
```

Comment:
How about using the error level?

```suggestion
        console.error('An unauthorized error occurred. Please check your API Key.');
```